tidalapi/page.py

```diff
@@ -99,17 +145,30 @@ def get(self, endpoint: str, params: Optional[Dict[str, Any]] = None) -> "Page":
         return self.parse(json_obj)
 
 
-class PageCategory(object):
+@dataclass
+class More:
+    api_path: str
+    title: str
+
+    @staticmethod
+    def from_response(json_obj: JsonObj) -> Optional["More"]:
```

Comment:
I prefer your naming convention, but the rest of the library uses `parse`, so I think we should here.

Reply:
Also why not a class method and then return `cls()` at the end?

Reply:
A class method would be fine here, too. I'll make these changes.